R/fix_c3_costs.R

```diff
@@ -6,7 +6,7 @@
 #' @return A data frame with costs applied
 #' @export
 #'
-#'@importFrom dplyr mutate case_when
+#' @importFrom dplyr mutate case_when
```

Comment:
Could you lose these imports and just do `dplyr::mutate(cost_total_net = dplyr::case_when(` please?